### ISSUE TITLE
duration output in before map is not believable

### ISSUE THREAD

**Issue by iankirkman** [tejasmhos/seattlecollision#3]
Check out the difference for 12 months versus anything less (even 11) in the duration toggle output for just the Before map. It does not appear reasonable.

We should probably look into the data creation and query creation specific to the days before calculation. I made an issue so we don't forget.

**Comment by alyssacolony** [comment]
I also noticed that the "during" and "after" for that same functionality seem as though they don't change as we adjust the input. Perhaps its all related.

**Comment by iankirkman** [comment]
I fixed a bug in the query string that should pull the correct before totals now, but I'm leaving this open because the marker sizes don't seem to change as much as expected. 12 months markers for 2016 base year should be approx twice as large as the 5 month markers. Assigning to Alyssa.

**Comment by iankirkman** [comment]
Current status (I think) is that Alyssa made the maps really pretty and distinguished the markers. We are gong to look into disabling the zoom and adding info about what marker colors mean. Any updates?

**Comment by alyssacolony** [comment]
I looked in to disabling the zoom and it isn't an option. Apparently it can be done in leaflet but not in folium. The other thing we talked about was maybe creating an interactive option to let the user choose a different map tile to better facilitate zooming. I am not a huge fan of this option as I think the map tile interaction would be slightly confusing (e.g., most users don't know what a map tile is. I didn't before this project) and doesn't really help with our use cases, which I think are more about the big picture. That said, it is definitely doable and we can do it if someone thinks its a priority. Let's discuss on Monday and we can make it happen on Tuesday if we decide it's important.
